### PR TITLE
pod-overlay example

### PR DESCRIPTION
**Pull request by zhaochun-ma**

### CODE REVIEW

File: advanced-configuration/pod-overlay/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Configure pod overlay
+- Create configmap used by pod overlay (required data: `key: pod-template.yaml`, `value: <yaml content>` which conform with kubernetes [StatefulSetSpec](https://github.com/kubernetes/api/blob/master/apps/v1beta1/types.go#L208) API). 
+yaml content must begin with `spec`(refer to `statefulset.spec`). Users are only allowed to configure fields inside `statefulset.spec.template`. If they try to set fields outside of it, the configuration will be considered as invalid.
+
+  - option 1: add a custom init container
+  
+    `kubectl -n confluent create configmap foo-pod-overlay --from-file=pod-template.yaml=$TUTORIAL_HOME/extra-init-container.yaml`
+  - option 2: enable QoS class of BestEffort
+  
+    `kubectl -n confluent create configmap foo-pod-overlay --from-file=pod-template.yaml=$TUTORIAL_HOME/enable-best-effort-qos.yaml`
```
Comment (domenicbove) [inline advanced-configuration/pod-overlay/README.md:66]: these look good, curious what are the top use cases again

**Review by domenicbove — APPROVED** [review]
LGTM!

File: advanced-configuration/pod-overlay/README.md (hunk 1 above)
Comment (zhaochun-ma) [reply advanced-configuration/pod-overlay/README.md:66]: these two use cases have associated FF tickets so I figured these would be top use cases and provided example here

**Comment by rohit2b** [comment]
LGTM!

**Review by rohit2b — APPROVED** [review]

**Pull request merged**